synapse/config/emailconfig.py

```diff
@@ -116,8 +116,9 @@ def read_config(self, config, **kwargs):
 
             if len(missing) > 0:
                 raise RuntimeError(
-                    "email.password_reset_behaviour is set to 'local' "
-                    "but required keys are missing: %s"
+                    "Password resets emails are configured to be sent from "
+                    "this homeserver due to a partially email block. "
```

Comment:
a partially email block?

In any case: we don't know that it has been configured due to the email block - it might be explicit config?

Reply:
Well, the explicit config is `email.trust_identity_server_for_password_resets`, so you'd get a partial email block anyways.